packages/appstore/src/components/cfd-account-type-details/cfd-account-type-details.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import cfd_account_details from 'Constants/cfd-account-type-details';
+
+type TCFDAccountTypeDetailsProps = {
+    platform: 'mt5' | 'dxtrade';
+    is_eu: boolean;
+    account_type: 'synthetic' | 'financial' | 'financial_stp';
+};
+
+type TCFDAccountTypeDetails = {
+    title: string;
+    description: string;
+    leverage: { getKey: () => string; getValue: () => string };
+    'margin-call': { getKey: () => string; getValue: () => string };
+    'stop-out-level': { getKey: () => string; getValue: () => string };
+    'number-of-assets': { getKey: () => string; getValue: () => string };
+};
+
+const CFDAccountTypeDetails = ({ platform, is_eu, account_type }: TCFDAccountTypeDetailsProps) => {
+    const [cfd_account_detail, setCfdAccountDetails] = React.useState<TCFDAccountTypeDetails>();
+
+    React.useEffect(() => {
+        if (account_type === 'financial') {
+            setCfdAccountDetails(
+                is_eu ? cfd_account_details[`${platform}`].financial_eu : cfd_account_details[`${platform}`].financial
```

Comment:
we should also handle `financial_au` if the client's location is Australia

Reply:
had missed this part while referring to previous code, now i have made the required changes